### PR TITLE
fix(metadata): Fix misaligned multiselect dropdown

### PR DESCRIPTION
**Pull request by swfree**
Issue: misaligned dropdown in Chrome (Mac), IE11 & Edge

**Mac Chrome - before**
mac_chrome_before

**Mac Chrome - after**
mac_chrome_after

after_mac_chrome_1

**Edge - before**
before_win_edge_1

**Edge - after**
after_win_edge_2

**IE11 - before**
before_ie11

**IE11 - after**
after_ie11

**Safari - after**
after_safari_1

**Firefox - after**
after_mac_firefox_2

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @swfree has signed the CLA. Thanks for the pull request!

File: src/components/select-field/__tests__/BaseSelectField.test.js [hunk 1]
```diff
@@ -242,6 +242,20 @@ describe('components/select-field/BaseSelectField', () => {
                 expect(overlay.hasClass(OVERLAY_SCROLLABLE_CLASS)).toBe(result);
             },
         );
+
+        test('should apply preventOverflow modifier when isEscapedWithReference is true', () => {
+            const wrapper = shallowRenderSelectField({ isEscapedWithReference: true });
+
+            const props = wrapper.find('PopperComponent').props();
+            expect(props.modifiers.preventOverflow).toBeDefined();
```
Comment (jpfranco) [inline src/components/select-field/__tests__/BaseSelectField.test.js]: nit: Can we use `toEqual` in order to also check that `isEscapedWithReference` is passed correctly?

**Comment by mickr** [comment]
I'm assuming that the other places where this worked already are still working?

**Comment by swfree** [comment]
@mickr yup! Verified that MultiSelectField and SingleSelectField uses elsewhere are behaving the same

**Pull request merged**